src/test/java/com/dougnoel/sentinel/webdrivers/WebDriverFactoryTest.java

```diff
@@ -0,0 +1,27 @@
+package com.dougnoel.sentinel.webdrivers;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class WebDriverFactoryTest {
+
+	@Test(expected = org.openqa.selenium.remote.UnreachableBrowserException.class)
```

Comment:
This is a good start, but you have some paths that aren't covered:
1.) Having a valid browserVersion (see below on how to to that.)
2.) Having a malformed URL and expecting an error.
3.) Having a driver returned.

You can see this in the code coverage report. I'd like to see 100% code coverage on this new file.